src/nbl/asset/utils/IShaderCompiler.cpp

```diff
@@ -238,7 +286,7 @@ IShaderCompiler::CCache::EntrySet::const_iterator IShaderCompiler::CCache::find_
             else
                 header = finder->getIncludeRelative(dependency.requestingSourceDir, dependency.identifier);
 
-            if (header.hash != dependency.hash || header.contents != dependency.contents)
+            if (header.hash != dependency.hash)
```

Comment:
You don't need `m_container.equal_range(mainFile);` and for loop here, since you have a `set` now instead of a `multi_set`
equal_range Is gonna return at most 1 here